### PR TITLE
[P1-N03] Replace all uint with uint256

### PR DESCRIPTION
**Pull request by chrismaree**
This PR replaces the use of `uint` through the codebase with `uint256`.

### CODE REVIEW

File: core/contracts/financial-templates/implementation/WETH9.sol [hunk 1]
```diff
@@ -63,17 +63,17 @@ contract WETH9 {
         return address(this).balance;
     }
 
-    function approve(address guy, uint wad) public returns (bool) {
+    function approve(address guy, uint256 wad) public returns (bool) {
```
Comment (nicholaspai) [inline core/contracts/financial-templates/implementation/WETH9.sol:66]: Who came up with these horrible variable names?

Comment (chrismaree) [reply core/contracts/financial-templates/implementation/WETH9.sol:66]: what? you dont want to `wad` your `guy`? is you `flip` not `flop`ed yet ya `shut?`

Comment (mrice32) [reply core/contracts/financial-templates/implementation/WETH9.sol:66]: ^^ Maker 🐳

**Comment by chrismaree** [comment]
> Should we also change all `int` to `int256`?

If we really want to be consistent, than yes. I have added this and pushed the changes.

**Pull request merged**